tests/vmi_configuration_test.go

```diff
@@ -2186,7 +2186,10 @@ var _ = Describe("[sig-compute]Configurations", decorators.SigCompute, func() {
 			Expect(err).ToNot(HaveOccurred())
 			vmiPod, err := libpod.GetPodByVirtualMachineInstance(vmi, vmi.Namespace)
 			Expect(err).NotTo(HaveOccurred())
-			defer storage.RemoveHostDiskImage(tmpHostDiskDir, vmiPod.Spec.NodeName)
+			defer func() {
+				err := storage.RemoveHostDisk(tmpHostDiskDir, vmiPod.Spec.NodeName)
```

Comment:
Same
`Expect(...).To(Succeed())`

But this test should be fixed as a follow up as `defer` is usually not a good fit.